docker-compose.yml

```diff
@@ -124,15 +124,13 @@ services:
       HASURA_GRAPHQL_IDLE_TIMEOUT: 180 # Idle connection timeout in seconds
       HASURA_GRAPHQL_TIMEOUT: 60 # Request timeout in seconds
       HASURA_GRAPHQL_ENABLE_QUERY_CACHE: true # Enable query caching
-      HASURA_GRAPHQL_PG_CONNECTIONS: 180 # Number of connections per PG connection pool
-      HASURA_GRAPHQL_PG_TIMEOUT: 180 # Query timeout in seconds
 
       # Subscriptions
       HASURA_GRAPHQL_ENABLE_SUBSCRIPTIONS: true # Enable subscriptions
       HASURA_GRAPHQL_ENABLE_WEBSOCKETS: true # Enable WebSockets
       HASURA_GRAPHQL_SUBSCRIPTIONS_PER_STRIPE: 10 # Subscriptions per stripe
       HASURA_GRAPHQL_SUBSCRIPTIONS_TIMEOUT: 180 # Subscription timeout in seconds
-      HASURA_GRAPHQL_WS_READ_COOKIE: true # Read cookies over WebSockets
+      # HASURA_GRAPHQL_WS_READ_COOKIE: true # Read cookies over WebSockets
```

Comment:
Why just comment out and not remove?

Reply:
We are not enforcing any CORS policy on Hasura at the moment, so if we enable this ENV without CORS policy, Hasura does not boot and return errors.

At the moment, my thought process goes like this:

If someone is smart enough to query our Hasura endpoint directly to get the data they need, it's much better than them querying the webapp and scraping it.
When we will have a dedicated API, and can rate limit endpoints or request, pair it with Bearer token, then we should enforce CORS on our Hasura endpoint and then activate this setting, and this could be useful if we start having table that have data that we want only visible to specific users.

I left it here to remember that this setting exist basically :)